src/ImageFitter/ImageFitter.cc

```diff
@@ -0,0 +1,227 @@
+/* This file is part of the CARTA Image Viewer: https://github.com/CARTAvis/carta-backend
+   Copyright 2018, 2019, 2020, 2021 Academia Sinica Institute of Astronomy and Astrophysics (ASIAA),
+   Associated Universities, Inc. (AUI) and the Inter-University Institute for Data Intensive Astronomy (IDIA)
+   SPDX-License-Identifier: GPL-3.0-or-later
+*/
+
+#define FWHM_TO_SIGMA 0.5 / sqrt(2 * log(2))
+#define DEG_TO_RAD M_PI / 180.0
+
+#include "ImageFitter.h"
+
+#include <omp.h>
+
+using namespace carta;
+
+ImageFitter::ImageFitter(std::string unit) {
+    _image_unit = unit;
+
+    _fdf.f = FuncF;
+    _fdf.df = NULL; // internally computed using finite difference approximations of f when set to NULL
```

Comment:
Use `nullptr` here and elsewhere in the code

Reply:
fixed.